imports/plugins/core/files/server/fileCollections.js

```diff
@@ -1,94 +1,80 @@
-import { MongoInternals } from "meteor/mongo";
 import { WebApp } from "meteor/webapp";
 import { check } from "meteor/check";
 import { Security } from "meteor/ongoworks:security";
 import fetch from "node-fetch";
-import Logger from "@reactioncommerce/logger";
 import {
   MeteorFileCollection,
   RemoteUrlWorker,
   TempFileStoreWorker
 } from "@reactioncommerce/file-collections";
-import { AbsoluteUrlMixin } from "/imports/plugins/core/core/server/Reaction/absoluteUrl";
 import { MediaRecords } from "/lib/collections";
-import setUpFileCollections from "./no-meteor/setUpFileCollections";
 
 /**
  * Functions and objects related to file upload, download, and storage
  * @namespace Files
  */
 
-const mongodb = MongoInternals.NpmModules.mongodb.module;
-const { db } = MongoInternals.defaultRemoteCollectionDriver().mongo;
-
-const {
-  downloadManager,
-  Media: NoMeteorMedia,
-  stores,
-  tempStore
-} = setUpFileCollections({
-  absoluteUrlPrefix: AbsoluteUrlMixin.absoluteUrl(),
-  db,
-  Logger,
-  MediaRecords: MediaRecords.rawCollection(),
-  mongodb
-});
-
-WebApp.connectHandlers.use("/assets/uploads", (req, res) => {
-  req.baseUrl = "/assets/uploads"; // tus relies on this being set, which is an Express thing
-  tempStore.connectHandler(req, res);
-});
-
 /**
- * @name Media
- * @type MeteorFileCollection
- * @memberof Files
- * @summary Defines the Media FileCollection
- *   To learn how to further manipulate images with Sharp, refer to
- *   {@link http://sharp.pixelplumbing.com/en/stable/|Sharp Docs}
- * @see {@link https://github.com/reactioncommerce/reaction-file-collections}
+ * @param {Object} context App context
+ * @return {undefined}
  */
-const Media = new MeteorFileCollection("Media", {
-  allowInsert: (userId, doc) => Security.can(userId).insert(doc).for(MediaRecords).check(),
-  allowUpdate: (userId, id, modifier) => Security.can(userId).update(id, modifier).for(MediaRecords).check(),
-  allowRemove: (userId, id) => Security.can(userId).remove(id).for(MediaRecords).check(),
-  check,
-  collection: MediaRecords,
-  DDP: Meteor,
-  allowGet: () => true, // add more security here if the files should not be public
-  stores,
-  tempStore
-});
+export default function meteorFileCollectionStartup(context) {
+  const { files } = context;
+  if (!files) throw new Error("Expected context.files");
 
-// For backward-compatibility with code relying on how CFS did it, we'll put a
-// reference to the backing MongoDB collection on Media.files property as well.
-Media.files = MediaRecords;
+  const { stores, tempStore } = files;
+  if (!stores) throw new Error("Expected context.files.stores");
+  if (!tempStore) throw new Error("Expected context.files.tempStore");
 
-WebApp.connectHandlers.use("/assets/files", downloadManager.connectHandler);
+  WebApp.connectHandlers.use("/assets/uploads", (req, res) => {
+    req.baseUrl = "/assets/uploads"; // tus relies on this being set, which is an Express thing
```

Comment:
`tus`?

Reply:
https://tus.io/

It's what file-collections package uses for uploading.